src/DataService/ObjectInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Lullabot\Mpx\DataService;
+
+/**
+ * Defines an interface object properties common to all mpx objects.
+ */
+interface ObjectInterface
+{
+    /**
+     * Returns The date and time that this object was created.
```

Comment:
"Returns the" instead of "Returns The". This is happening too in the following comments of properties